website/screens/principles/iconography/usage/IconographyUsagePage.tsx

```diff
@@ -0,0 +1,97 @@
+import DocFooter from "@/common/DocFooter";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import { DxcBulletedList, DxcFlex } from "@dxc-technology/halstack-react";
+import Image from "@/common/Image";
+import icons from "./images/Iconography_image.png";
+import Link from "next/link";
+
+const sections = [
+  {
+    title: "Usage",
+    content: (
+      <>
+        <Image src={icons} height={400} alt="Halstack Icons" />
```

Comment:
I think is better not to set the height in this case.

Reply:
Changed